### ISSUE TITLE
missingJsdocRule false positive

### ISSUE THREAD

**Issue by bootrider** [Microsoft/tslint-microsoft-contrib#286]
a file with the code:

```
/**
 * Index.ts file
 */

export * from './app.component';
export * from './app.module';
```

does not show any error.

a file with the code

```
/**
 * test.ts file
 */

export * from './app.component';
export * from './app.module';
```

presents the following error:
`src/app/test.ts[4, 1]: File missing JSDoc comment at the top-level: src/app/test.ts`

if I change the test.ts file to:

```
/**
 * test.ts file
 */

export * from './app.component';

/**
 * test.ts file
 */
export * from './app.module';
```

the error dissappear

**Comment by HamletDRC** [comment]
@bootrider I cannot get your example to fail. 

The rule is based on a regex looking for /** at the beginning of a line: 

    /^\/\*\*\s*$/gm

Do you perhaps have a trailing space after the second * character in the first line?

**Comment by HamletDRC** [comment]
Can you possibly upload the file somewhere so that I can see the whitespace and file encoding?

**Comment by HamletDRC** [comment]
@bootrider are you still experiencing this issue? I cannot reproduce it.

**Comment by HamletDRC** [comment]
I am closing the issue because I cannot reproduce :(
